.pre-commit-config.yaml

```diff
@@ -12,18 +12,18 @@ repos:
       - id: check-yaml
       - id: check-added-large-files
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 4.0.1
-    hooks:
-      - id: flake8
-        name: Check flake8
-
   - repo: https://github.com/psf/black
     rev: 21.12b0
     hooks:
       - id: black
         name: Check black
 
+  - repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        name: Check flake8
+
```

Comment:
Seems like you just moved it from one place to another ? Not sure this should be part of the commit.

Reply:
It usually worth to checking the commits for more info: https://github.com/IoT-Inspector/unblob/pull/236/commits

He did it directly to execute black prior to flake8.